Dep Installer: update documentation (#853)
The-OpenROAD-Project/OpenROAD-flow: docs/user/BuildWithDocker.md

## Clone and Build

> ### Build image generally

Reviewer: ```suggestion
### Build Docker image
```

> The following instructions build the docker image with a parameterizied OS in two stages. 
> Dev stage for installing all the dependencies and packages that are needed to run OpenRoad and OpenRoad Flow Scripts.
> Build stage actually builds all of the binaries needed to run flows.

Reviewer: ```suggestion
The following instructions build the docker image with a parameterized OS in two stages. 
Dev stage for installing all the dependencies and packages that are needed to run OpenROAD and OpenROAD Flow Scripts.
The build stage actually builds all of the binaries needed to run flow (i.e., `openroad` and `yosys`).
```

``` shell
git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts
cd OpenROAD-flow-scripts
./etc/DockerHelper.sh create -target=dev -os=$OS_NAME
./etc/DockerHelper.sh create -target=builder -os=$OS_NAME
```

> ### Build image with default values
> 
> The following instructions build the docker image with Centos 7 as base OS:
> 

Reviewer: ```suggestion
### Build image with default values

The following instructions build the docker image with Centos 7 as the base OS:

```
